client-src/elements/chromedash-guide-new-page.ts

```diff
@@ -9,8 +9,11 @@ import {
 import {ALL_FIELDS} from './form-field-specs';
 import {SHARED_STYLES} from '../css/shared-css.js';
 import {FORM_STYLES} from '../css/forms-css.js';
-import {setupScrollToHash, formatFeatureChanges} from './utils';
+import {setupScrollToHash, formatFeatureChanges, FieldInfo} from './utils';
```

Comment:
Looks like imports are out of order here (and they were before you started even) 😅

Reply:
ok let me ruin organize imports before merging